closure/templates/closure_js_template_library.bzl

```diff
@@ -32,6 +32,8 @@ def _impl(ctx):
         args += ["--bidiGlobalDir=%s" % ctx.attr.bidi_global_dir]
     if ctx.attr.plugin_modules:
         args += ["--pluginModules=%s" % ",".join(ctx.attr.plugin_modules)]
+    for arg in ctx.attr.defs:
```

Comment:
could you add validation similar to jscompiler one?

Reply:
good catch, added!